scalameta/semantic/shared/src/main/scala/scala/meta/semantic/v1/Location.scala

```diff
@@ -22,23 +26,34 @@ case class Location(addr: Address, start: Int, end: Int) {
 }
 
 @root trait Address {
+  def toProto: proto.Address
```

Comment:
I wonder whether we want to have serialization methods directly in domain objects.